docs/products/access/topics/oidc/claims-lifecycle/claims-lifecycle.md

```diff
@@ -0,0 +1,76 @@
+# User claims and attributes lifecycle
+
+This section aims to explain and visualise how Access deals with user claims/attributes during the most commonly used flows.
+
+## Authorization
+
+![AWS Configuration](img/access-claims-lifecycle-authorization.jpg)
+
+1. The OAuth Client / OIDC Relying Party initiates the authorisation code flow.
+2. Based on the _client_id_ parameter (optionally also _idp_id_), Access resolves the IdentityProvider instance that will be used for user
+   authentication.
+    1. If the underlying IdentityProvider is of type OIDC (e.g. Tulip), Access redirects the user to the defined OIDC Provider with another
+       authorization request.
+        1. Once authenticated, the OIDC Provider redirects back to Access with the authorisation code and Access exchanges the code for the
+           AccessToken / IDToken pair. The IDToken is the primary source of information about the user. An internal User Object is created
+           and populated with all the user claims available in the IDToken. In addition to the user attributes, the AMR claim is also stored
+           in the User Object.
+        2. If the IdentityProvider is of type Tulip, it is likely that a User Data Enhancer (UDH) API integration is configured. If this is
+           the case, the user claims are requested via another API call from UDH, Access passes the user identifier (subject) and a set of
+           scopes requested by the client. Otherwise, the User Object is persisted (see #2.a.5).
+        3. The UDH response is mapped to a UserInfo standard format.
+        4. The User Object is populated with any claims returned by the UDH service. If a given claim already exists in the user object,
```

Comment:
Actually with current implementation we don't combine UDH claims with ID Token claims.
We just map the UDH result to User Object, and from ID Token we only take AMR and ACR claims.

Reply:
I will mention it in the description 👍